EXILED/Exiled.Events/Handlers/Player.cs

```diff
@@ -85,6 +91,7 @@ public class Player
         /// <remarks>
         /// Invoked after <see cref="UsingItem"/>, if a player's class has
         /// changed during their health increase, won't fire.
+        /// Items that will invoke this event: SCP-1853, SCP-207, Anti-Cola, all medical, SCP-244, SCP-1576, SCP-268 and all candies.
```

Comment:
also not necessary as all items trigger this

Reply:
Same as what I said above

Reply:
same as above